src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,73 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 48px;
+  justify-content: center;
+  grid-auto-flow: row;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $card-height;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: repeat(2, $card-width);
+    gap: 48px;
+    justify-content: center;
+    grid-auto-flow: row;
+
+    &__card {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      height: $card-height;
+      border: 1px solid #ccc;
+      border-radius: 5px;
+    }
+  }
+}
```

```suggestion
.catalog {
  display: grid;
  grid-template-columns: repeat(1, $card-width);
  gap: 48px;
  justify-content: center;
  grid-auto-flow: row;

  &__card {
    display: flex;
    justify-content: center;
    align-items: center;
    height: $card-height;
    border: 1px solid #ccc;
    border-radius: 5px;
  }
  
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, $card-width);
   }
}
```
DRY, fix it everywhere